### PR TITLE
update eg.csv list

### PR DESCRIPTION
**Pull request by moeltaher**
add new 6 websites

### CODE REVIEW

File: lists/eg.csv [hunk 1]
```diff
@@ -684,3 +684,10 @@ https://www.proxyswitcher.com,ANON,Anonymization and circumvention tools,2018-02
 http://astrill.com,ANON,Anonymization and circumvention tools,2018-02-01,AFTE,
 http://www.vpntool.com,ANON,Anonymization and circumvention tools,2018-02-01,AFTE,
 https://www.zeronet.io,ANON,Anonymization and circumvention tools,2018-02-25,AFTE,
+https://www.sportsinteraction.com,ANON,Anonymization and circumvention tools,2018-03-26,AFTE,
```
Comment (sneft) [inline lists/eg.csv]: Should this be GMB (Gambling)?

Comment (moeltaher) [reply lists/eg.csv]: Yes, it's my mistake

Comment (moeltaher) [reply lists/eg.csv]: @sneft i updated eg.csv and corrected the category
https://github.com/citizenlab/test-lists/pull/321/commits/1d979647db68841df8213d4ecd089c0faf143bc9

**Comment by hellais** [comment]
lgtm

**Pull request merged**